easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.4.4-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,46 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Ake Sandgren
+# HPC2N
+# Umea University
+
+name = 'MotionCor2'
+version = '1.4.4'
+
+homepage = 'https://msg.ucsf.edu/'
+description = """MotionCor2 correct anisotropic image motion at the
+single pixel level across the whole frame, suitable for both single
+particle and tomographic images. Iterative, patch-based motion detection
+is combined with spatial and temporal constraints and dose weighting.
+
+Cite publication: Shawn Q. Zheng, Eugene Palovcak, Jean-Paul Armache,
+Yifan Cheng and David A. Agard (2016) Anisotropic Correction of
+Beam-induced Motion for Improved Single-particle Electron
+Cryo-microscopy, Nature Methods, submitted.
+BioArxiv: https://biorxiv.org/content/early/2016/07/04/061960
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+sources = [{
+    'filename': '%(name)s_%(version)s.zip',
+    'extract_cmd': 'unzip -j %s'
+}]
+checksums = ['128f5cdf83b5023204c10b4801baa13f0a56a950e0bc6e0ed26c7f3aa1df59d7']
+
+# No longer directly downloadable, available from https://msg.ucsf.edu/software
+# CUDA is a build dependency to make sure it gets installed.
+# It's actually a runtime dependency, but that's handled in the wrapper to
+# make sure it doesn't conflict with whatever toolchain happens to be loaded.
+# Change CUDA version to match the latest one used in this version
+# of MotionCor2
+builddependencies = [
+    ('binutils', '2.35'),
+    ('CUDAcore', '11.1.1', '', True),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libjpeg-turbo', '2.0.5'),
+    ('LibTIFF', '4.1.0'),
```

Comment:
@ThomasHoffmann77 you removed the `XZ` and `jbigkit` dependencies, you have a good reason for that?
otherwise I would just keep them, I think it makes sense to have them available for `MotionCor2`.

Reply:
And we should get that motioncor2 easyblock merged so that the sources line can be restored to the simple version it has been.

Reply:
> @ThomasHoffmann77 you removed the `XZ` and `jbigkit` dependencies, you have a good reason for that? otherwise I would just keep them, I think it makes sense to have them available for `MotionCor2`.

@smoors If I interpret the output of ldd correctly, the MotionCor2 binary neither uses liblzma nor libjbig. That's why I removed both dependencies. What would be a reason to keep them?

Reply:
Yeah, it really only needs LibTIFF, the rest can be dropped. I can't remember why I added them in the first place any longer.

Reply:
I just downloaded the binary and `libjbig` is definitely present for me (liblzma used to be present in a previous version, but is gone now):
```
ldd MotionCor2_1.4.4_Cuda113-08-11-2021
        linux-vdso.so.1 =>  (0x00007ffd1fb82000)
        libcufft.so.10 => not found
        libcudart.so.11.0 => not found
        libcuda.so.1 => not found
        libnvToolsExt.so.1 => not found
        libtiff.so.5 => /lib64/libtiff.so.5 (0x00007f2527e51000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f2527c35000)
        librt.so.1 => /lib64/librt.so.1 (0x00007f2527a2d000)
        libdl.so.2 => /lib64/libdl.so.2 (0x00007f2527829000)
        libstdc++.so.6 => /lib64/libstdc++.so.6 (0x00007f2527521000)
        libm.so.6 => /lib64/libm.so.6 (0x00007f252721f000)
        libc.so.6 => /lib64/libc.so.6 (0x00007f2526e51000)
        libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007f2526c3b000)
        libjbig.so.2.0 => /lib64/libjbig.so.2.0 (0x00007f2526a2f000)
        libjpeg.so.62 => /lib64/libjpeg.so.62 (0x00007f25267da000)
        libz.so.1 => /lib64/libz.so.1 (0x00007f25265c4000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f25282d5000)
```

Reply:
@smoors As soon as I load LibTIFF/4.1.0-GCCcore-10.2.0, in the ldd output  libjpeg changes from .so.62 to .so.8 and libjbig disappears. Why?

Reply:
ok, so my system (centos7) libtiff depends on libjbig, but with more recent libtiff this is no longer the case.
however, more recent libtiff depends on liblzma, but XZ is not a dependency of LibTIFF, which is bug.
I'll make a PR to fix this.

Reply:
Yeah, MotionCor2 binary ONLY loads libtiff. What LibTIFF uses is out-of-scope for MotionCor2

Yuo need to use objdump -x to see what the primary libs it needs are.

Reply:
or use `ldtree`.

Reply:
PR https://github.com/easybuilders/easybuild-easyconfigs/pull/14079